packages/generate-bot/templates/src/{{programName}}.ts

```diff
@@ -12,26 +12,42 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {Application} from 'probot';
+import {Probot} from 'probot';
+import {logger} from 'gcf-utils';
 
 const CONFIGURATION_FILE_PATH = '{{programName}}.yml';
 
 interface Configuration {
   randomBoolean: boolean;
 }
 
-export = (app: Application) => {
-  app.on(['issues.opened', 'pull_request.opened'], async context => {
+export = (app: Probot) => {
+  app.on(['pull_request.opened'], async context => {
     const config = (await context.config(
       CONFIGURATION_FILE_PATH,
       {}
     )) as Configuration;
 
     if (
-      (context.payload.pull_request || context.payload.issue) &&
```

Comment:
Typescript was complaining about types here. In general, we probably don't want to mix these types of events in the same handler function -- I broke the issue event into a separate, nearly identical handler to demonstrate to users that you could have more than one.